resources/js/features/root/pages/AdminUsersList.page.tsx

```diff
@@ -14,6 +14,12 @@ export default (() => {
         translations="tables.tables.users"
         additionalColumns={["actions"]}
         columnOptions={{
+          id: {
+            metaParams: {canControlVisibility: false},
+          },
+          name: {
+            metaParams: {canControlVisibility: false},
+          },
           email: {
```

Comment:
wydaje mi się, że prędzej czy później warto zrobić jakąś abstrakcję na te wszystkie opcje, inaczej będzie niezły bałagan z propsami. Najlepiej, jakby ten obiekt columnOptions tworzyć poza komponentem, albo w memo w komponencie za pomocą jakiejś funkcji

Reply:
Chodzi o takie helpery jak na przykład tanstack table ma, że się pisze `h.accessor(name,{...})`? Może tak, ale na razie to jest i tak fejkowy backend więc bym to zostawił na czas jak ta strona będzie finalizowana z prawdziwymi danymi.